docs/explanations/audit-scanner.md

```diff
@@ -0,0 +1,273 @@
+  rule: testing-cap-policy
+  source: kubewarden
+  timestamp:
+    nanos: 0
+    seconds: 1689017140
+...
+summary:
+  error: 0
+  fail: 0
+  pass: 6
+  skip: 0
+  warn: 0
+```
+
+
+### Namespace-Specific Audit Results example
+
+In this example, the audit scanner has evaluated multiple policies on resources
+within the `default` namespace. The results indicate that some of the resources
+failed the validation for the `cap-no-privilege-escalation` policy, while
+others passed the validation for the `cap-do-not-run-as-root` policy. The
+`summary` section shows a summary of the audit results, indicating the number
+of failures and passes.
+
+```yaml
+apiVersion: wgpolicyk8s.io/v1beta1
+kind: PolicyReport
+metadata:
+  creationTimestamp: "2023-07-10T19:28:05Z"
+  generation: 4
+  labels:
+    app.kubernetes.io/managed-by: kubewarden
+  ...
+results:
+- message: one of the containers has privilege escalation enabled
+  policy: cap-no-privilege-escalation
+  ...
+  resourceSelector: {}
+  resources:
+  - apiVersion: apps/v1
+    kind: Deployment
+    name: nginx-deployment
+    namespace: default
+    ...
+  result: fail
+  rule: no-privilege-escalation
+  source: kubewarden
+  timestamp:
+    nanos: 0
+    seconds: 1689017383
+- policy: cap-do-not-run-as-root
+  ...
+  resourceSelector: {}
+  resources:
+  - apiVersion: apps/v1
+    kind: Deployment
+    name: nginx-deployment
+    namespace: default
+    ...
+  result: pass
+  rule: do-not-run-as-root
+  source: kubewarden
+  timestamp:
+    nanos: 0
+    seconds: 1689017383
+...
+summary:
+  error: 0
+  fail: 8
+  pass: 10
+  skip: 0
+  warn: 0
+```
+
+
+By using the audit scanner and analyzing the generated audit reports, you can
+gain insights into the compliance of your resources with the deployed policies,
+helping you ensure that your cluster remains secure and adheres to the defined
+rules and validations.
+
+---
+
+## Policies
+
+Policies will be evaluated by the audit scanner by default. Operators that want to
+skip a policy evaluation in the Audit scanner should set `false` to the
+`backgroundAudit` field in the policy spec. Furthermore,  policies in
+Kubewarden now support two optional annotations:
+`io.kubewarden.policy.severity` and `io.kubewarden.policy.category`. These
+annotations can be defined in the `metadata.yml` file associated with a policy.
+
+- The `io.kubewarden.policy.severity` annotation allows you to specify the
+  severity level of the policy violation, such as "critical", "high", "medium",
+  "low" or "info".
+- The `io.kubewarden.policy.category` annotation allows you to categorize the
+  policy based on a specific domain or purpose, such as "PSP",
+  "compliance", or "performance".
+
+See the policy authors [docs](../writing-policies/index.md) for more info.
+Furthermore, the `metadata.yaml` file in one of the [template
+projects](https://github.com/kubewarden/rust-policy-template/blob/main/metadata.yml)
+can be a nice example how to configure the policy
```

Comment:
I would remove the line "Furthermore, the `metadata.yaml` (...)". That's info for policy authors. To configure a policy, one does `kwctl scaffold` and changes whatever one wants.